containers/release-candidate/src/daffodil-release-candidate

```diff
@@ -46,6 +46,25 @@ Options:
 USAGE
 }
 
+# Function to get a non-empty string as input
+get_non_empty_input() {
+    local prompt="$1" # prompt the user with this string
+    local input
+
+    while true; do
+        # Prompt the user for input
+        read -p "$prompt" input
+
+        # Check if the input is non-empty
+        if [[ -n $input ]]; then
+            echo "$input"
+            return 0
+        else
+            echo "Empty input. Please try again."
```

Comment:
Does the "empty input" message need to go to stderr? Otherwise I think it will become part of resulting variable.